### PR TITLE
Check if using discovery servers (#488)

### PR DESCRIPTION
**Pull request by brianhlin**
My java's pretty weak (don't think I could write a good test, sorry) but this seems like a simple fix!

### CODE REVIEW

File: src/main/java/com/nutomic/syncthingandroid/syncthing/RestApi.java [hunk 1]
```diff
@@ -440,9 +440,14 @@ protected void onPostExecute(String s) {
                     SystemInfo si = new SystemInfo();
                     si.alloc = system.getLong("alloc");
                     si.cpuPercent = system.getDouble("cpuPercent");
-                    si.extAnnounceTotal = system.getInt("discoveryMethods");
-                    si.extAnnounceConnected =
+                    if system.has("discoveryEnabled") {
```
Comment (Nutomic) [inline src/main/java/com/nutomic/syncthingandroid/syncthing/RestApi.java]: Java requires parentheses around the if condition: `if (system.has("discoveryEnabled"))`. Add that and I'll merge it ;)

**Comment by brianhlin** [comment]
Did I mention that my java-fu is weak? :)

**Comment by Nutomic** [comment]
I noticed that :p 

Can you also squash your commits please?

**Comment by brianhlin** [comment]
How's that?

**Comment by Nutomic** [comment]
@st-jenkins please build (let's see how this works)

**Comment by Nutomic** [comment]
@st-jenkins build this

**Comment by Nutomic** [comment]
Hm guess the bot isn't in the mood...

Merged, thank you :)

**Pull request merged**